.github/workflows/unit.js.yml

```diff
@@ -0,0 +1,34 @@
+# This workflow will do a clean install of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Unit tests
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  unit_test:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x]
+        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
```

Comment:
Node.js is only used as a build dependency, right? In that case we don’t need to support multiple Node.js versions —  we can always just use the one we want (i.e. the most recent stable). So let’s simplify by dropping `matrix` and just setting `node-version: 14` below?

Reply:
done